kolibri/core/assets/src/views/core-table.vue

```diff
@@ -37,8 +37,6 @@
   .core-table
     width: 100%
     font-size: 14px
-    // without this, we get inconsistent column widths
-    table-layout: fixed
```

Comment:
@christianmemije would you take a look at the changes to the tables?

I wasn't sure what the intended behavior of `table-layout: fixed` was, but after a brief inspection the tables look better without it?

Reply:
cc @DXCanas

Reply:
okay the goal was to get the two tables on the class roster page to line up with each other. However I think it has too many other negative consequences so let's maybe live with that for now